templates/windows/check_hash.ps1.epp

```diff
@@ -0,0 +1,17 @@
+#file computes the 512SHA for a given string and writes it to a file
```

Comment:
Shouldnt the corresponding `.erb` files now be removed? like by @Ramesh7 [here](https://github.com/puppetlabs/puppetlabs-sqlserver/pull/436/files)

Reply:
DONE